.github/workflows/ci.yml

```diff
@@ -6,26 +6,41 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
-        java: [16]
+        java: [ 11, 16 ]
```

Comment:
Doesn't 1.17 require Java 16? Or is this for compatibility with the 1.12 branch?

Reply:
This should just be Java 16

Reply:
```suggestion
        java: [ 16 ]
```